### DISCUSSION TITLE
Print file in Atom Editor

### DISCUSSION THREAD

**Discussion by kumar007git** [atom/atom#22303] [Q&A]
How can I print the contents of a file I open in Atom Editor? Is there a built-in command or a package I could use? Thanks!

**Comment by ThatXliner** [comment]
You would just open a file in Atom-

Or, if you're on Linux/MacOS:

```bash
cat path/to/file.txt
```
In your terminal

**Reply by UziTech** [reply]
Lol. I think they meant print to a printer. Like on paper.

**Comment by UziTech** [comment]
There are many [packages](https://atom.io/packages/search?q=Print) with different options.
